core/trino-main/src/test/java/io/trino/operator/output/BenchmarkPartitionedOutputOperator.java

```diff
@@ -612,6 +622,14 @@ private static void pollute()
                 data.setupData(null);
                 data.setPageCount(1);
                 benchmark.addPage(data);
+                // pollute row-wise processing
```

Comment:
this should go to commit that adds row-wise benchmark. BTW: do you know if it works?

Reply:
there is no explicit row-wise benchmark. It has to be set up manually by choosing `positionCount` ~= `partitionCount`.

And yes, row-wise code path pollution works, it has visible influence on benchmark results:
```
### with row-wise pollution
Benchmark                                   (channelCount)  (enableCompression)  (nullRate)  (partitionCount)  (positionCount)  (type)  Mode  Cnt    Score    Error  U
BenchmarkPartitionedOutputOperator.addPage               1                false           0               512              512  BIGINT  avgt   10  167.598 ± 12.291  ms/op

### without row-wise pollution
Benchmark                                   (channelCount)  (enableCompression)  (nullRate)  (partitionCount)  (positionCount)  (type)  Mode  Cnt    Score    Error  Units
BenchmarkPartitionedOutputOperator.addPage               2                false           0               512              512  BIGINT  avgt   10  213.097 ± 14.498  ms/op
```